packages/kit-bg/src/dbs/local/LocalDbBaseContainer.ts

```
@@ -59,9 +60,18 @@ export abstract class LocalDbBaseContainer implements ILocalDBAgent {
   async getRecordById<T extends ELocalDBStoreNames>(
     params: ILocalDBGetRecordByIdParams<T>,
   ): Promise<ILocalDBGetRecordByIdResult<T>> {
-    if (this.isCachedStoreName(params.name)) {
+    // eslint-disable-next-line prefer-const
+    let shouldUseCache = this.isCachedStoreName(params.name);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'shouldUseCache' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)